### ISSUE TITLE
Revise how we bootstrap (aka import) content for development

### ISSUE THREAD

**Issue by jonfroehlich** [makeabilitylab/makeabilitylabwebsite#420]
TL;DR: we need a new way of loading test content (talks, pubs, projects, news, etc.) into the dev environment. 

Currently, we have some awesome content bootstrapping code (originally written by @Jamjarman and then updated by me) that parses the bibtex and XML files on my UMD website and then uses that to auto-load content into the Django site. We originally wrote this to avoid the pain of manually loading data into the production website but it's become a more useful tool for loading your development environment with data. 

However, this will need to change--for two big reasons: (1) my UMD website is going away. Yes, I could host the same data on my CS UW website--and perhaps I will; however, see second point; (2) I would prefer not to continue to maintain two backend sources of data (my academic website and the ML website) and I would like the ML website to be the sole source of data. 

So, I think future bootstrapping code should actually simply download some content from the production site and load them into the database (e.g., the 5 most recent papers, news items, talks, etc.). I'm not sure what the best way is to do this...

**Comment by johnsonkuang** [comment]
We culd use a REST API to put all the data on the production site into a JSON format and then have individual developers then query the API and retreive that data and build a test database in that way.

**Comment by jonfroehlich** [comment]
Oh yes, I like that. And we need to build out that API anyway.

**Comment by jonfroehlich** [comment]
Despite @johnsonkuang's work on this, it's still not working. Need to keep testing and build test app that actually uses API (perhaps test app will be my webpage). Moving out of milestone for now.

**Comment by jonfroehlich** [comment]
@johnsonkuang said he's going to look into this.

**Comment by jonfroehlich** [comment]
@johnsonkuang, have you been able to work on this as expected? If not, I'm going to unassign you.

**Comment by johnsonkuang** [comment]
I’ve been working on it, will push the code once I’m finished.

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by jonfroehlich** [comment]
Great! Do you have an ETA?

**Comment by jonfroehlich** [comment]
Coming back up to say that Option 2 is best.

**Comment by jonfroehlich** [comment]
Talk to @mechanicjay about how to get a PostgreSQL dump of the -test server + copy all of the static files

**Comment by jonfroehlich** [comment]
To copy files from host to a Docker container, see: https://www.arungudelli.com/tutorial/docker/copy-files-folders-from-host-to-docker-container/

**Comment by jonfroehlich** [comment]
There is actually some functionality to export/import a database from Django. See this discussion, for example: https://stackoverflow.com/questions/7002194/how-to-copy-database-in-use-to-other-database-in-django

Would have to have Jason's help.

**Comment by jonfroehlich** [comment]
I'm trying locally in my WSL2 environment now. When you create bash shell into container, you need to be sudo to run the command:

```
apache@43cd75e1d80c:/code$ python manage.py dumpdata > dumped_data.json
bash: dumped_data.json: Permission denied
apache@43cd75e1d80c:/code$ su
Password:
root@43cd75e1d80c:/code# python manage.py dumpdata > dumped_data.json
root@43cd75e1d80c:/code#
```

**Comment by jonfroehlich** [comment]
OK, I wrote up a [Wiki page on how to do this](https://github.com/makeabilitylab/makeabilitylabwebsite/wiki/Importing-data). Hopefully, @mechanicjay and I can test soon.
